tests/callbacks/test_callbacks.py

```diff
@@ -107,7 +107,7 @@ def test_multiple_init_and_close(self, cb_cls: Type[Callback], dummy_state: Stat
 
 
 @pytest.mark.parametrize('cb_cls', get_cbs_and_marks(callbacks=True, loggers=True, profilers=True))
-@pytest.mark.parametrize('grad_accum', [1, 2])
+@pytest.mark.parametrize('grad_accum,remote', [(1, False), (2, False), pytest.param(1, True, marks=pytest.mark.remote)])
```

Comment:
remote is not used in this class?

Reply:
This is a bit confusing, but it is used to parameterize the test markers. The value of the variable is unused. Added a comment to explain this.